ocaml/testsuite/tests/typing-layouts-err-msg/value.ml

```diff
@@ -2,7 +2,6 @@
    flags = "-extension layouts_alpha"
    * expect
 *)
-(* CR layouts v2.9: all error messages below here are not reviewed for quality *)
```

Comment:
The error for the example on line 292 looks off:
```ocaml
let f: 'a. 'a -> ('b : void) = fun x -> x
[%%expect{|
Line 1, characters 40-41:
1 | let f: 'a. 'a -> ('b : void) = fun x -> x
                                            ^
Error: This expression has type ('a : value)
       but an expression was expected of type ('b : void)
       The layout of 'b is void, because
         of the annotation on the type variable 'b.
       But the layout of 'b must overlap with value, because
         it's an unannotated universal variable.
|}];;
```
In particular, the last line says "it's an unannotated universal variable".  That's true about `'a`, but not about `'b`, which is the referent of the "it's".

Reply:
I changed it to say `it is or unifies with an unannotated universal variable`. An alternative would be to produce different messages depending on the placement, but the complexity doesn't seem worth it since it's just for this one message type.

Reply:
Fine by me.